framework/include/systems/NonlinearSystemBase.h

```diff
@@ -274,9 +274,9 @@ class NonlinearSystemBase : public SystemBase, public PerfGraphInterface
   /**
    * Finds the implicit sparsity graph between geometrically related dofs.
    */
-  void
-  findImplicitGeometricCouplingEntries(GeometricSearchData & geom_search_data,
-                                       std::map<dof_id_type, std::vector<dof_id_type>> & graph);
+  void findImplicitGeometricCouplingEntries(
+      GeometricSearchData & geom_search_data,
+      std::unordered_map<dof_id_type, std::vector<dof_id_type>> & graph);
```

Comment:
Since we think this is only used internally, and since we're about to really mess with anyone crazy enough to be calling it from third-party code, should we make that policy official and move this to `protected:` (or even `private:`??) rather than `public:`, while we're at it?

Reply:
I like that plan!